config/settings/base.py

```diff
@@ -562,3 +562,8 @@
 # Shell Plus
 # ------------------------------------------------------------------------------
 SHELL_PLUS_PRINT_SQL_TRUNCATE = env.int("SHELL_PLUS_PRINT_SQL_TRUNCATE", default=10_000)
+
+# Endpoints
+TX_SERVICE_ALL_TXS_ENDPOINT_LIMIT_TRANSFERS = env.int(
+    "TX_SERVICE_ALL_TXS_ENDPOINT_LIMIT_TRANSFERS", default=1_000
```

Comment:
No strong opinion here bu maybe would be better default 0 and add logic to don't limit by default.

Reply:
Thinking on our providers, I think it's better to do it as the default. Also adding logic for not limiting would imply modifying the code way more